docs/configuring.md

```diff
@@ -0,0 +1,71 @@
+---
+id: configuring
```

Comment:
pls remove this metadata stuff from all the docs - it is not really used by mkdocs

and every document must start with the first level header - I'll add md linter here later - you can use `title` metadata as the first level title here